packages/kbn-eslint-config/typescript.js

```diff
@@ -97,6 +93,18 @@ module.exports = {
                 match: false,
               },
             },
+            {
+              selector: 'import',
```

Comment:
ignore incorrect casing for imports like `import {MY_CONSTANT} from './constants'`